### PR TITLE
Modified iis-app-create to handle different app-pool users

### PR DESCRIPTION
**Pull request by kshanafelt**
Now handles different app-pool users
Fixed bugs noted here: https://library.octopusdeploy.com/step-template/actiontemplate-iis-application-create
Also added paramenters to make its UI a closer match to _IIS web site and application pool_  deployment feature.

### CODE REVIEW

File: step-templates/iis-app-create.json [hunk 1]
```diff
@@ -3,14 +3,26 @@
...
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.NuGetFeedId": null,
+    "Octopus.Action.Package.NuGetPackageId": null
   },
-  "SensitiveProperties": {},
   "Parameters": [
     {
+      "Name": "CreateOrUpdateIisApp",
+      "Label": "Create or update an IIS virtual application and Application Pool",
+      "HelpText": "If true, sets the application to enable preloading.",
```
Comment (alfhenrik) [inline step-templates/iis-app-create.json]: Help text is incorrect

File: step-templates/iis-app-create.json [hunk 2]
```diff
@@ -79,13 +73,85 @@
...
+      "Name": "ApplicationPoolPassword",
+      "Label": "Application Pool Password",
+      "HelpText": "The password for the custom account given above.",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
     }
   ],
-  "LastModifiedBy": "ekrapfl",
```
Comment (alfhenrik) [inline step-templates/iis-app-create.json]: If you add this back in and modify them with your GitHub username, it will be shown on the library site

**Comment by kshanafelt** [comment]
@alfhenrik Made the modification you suggested. Thatnks.

**Comment by alfhenrik** [comment]
Thank you @kshanafelt this has been merged and pushed to production!

**Pull request merged**